creator-node/src/routes/tracks.js

```diff
@@ -94,6 +94,27 @@ module.exports = function (app) {
     })
   )
 
+    /**
+     * Delete all temporary transcode artifacts from track transcode handoff flow.
+     * This is called on the node that was handed off the transcode to clear the state from disk
+     */
+  app.post(
+    '/clear_transcode_and_segment_artifacts',
+    ensureValidSPMiddleware,
+    handleResponse(async (req, res) => {
+      const fileDir = req.body.fileDir
+      req.logger.info("Clearing filesystem fileDir", fileDir)
+      if (!fileDir.includes('tmp_track_artifacts')) {
+        return errorResponseBadRequest(
+          'Cannot remove track folder outside temporary track artifacts'
+        )
+      }
+      await removeTrackFolder({ logContext: req.logContext }, fileDir)
```

Comment:
we should probably wrap this in a try/catch. if this part fails, then we wouldnt be returning any errorResponse, and only an err is thrown

Reply:
that's what handleResponse does, it wraps this entire function in a try catch

Reply:
yeah, but I think itd be nicer if we have a specific err thrown VS the general internal server error. not a huge deal but would give more context on the err

example: 
```
  app.get(
    '/test_err',
    handleResponse(async (req, res) => {
      throw new Error('test throwing in handleResponse')
    })
  )
```

server:
```
// 20220701161107
// http://35.192.87.132:4000/test_err

{
  "error": "Internal server error"
}
```
```
```